### PR TITLE
Improve feedback message for new command success

### LINKED ISSUE #45: Better feedback on `new`
## Problem

No message is displayed when `new` is successfully done.

## Goal

1. Let the user know once `new` is successfully done (or failed).
2. Possibly, indicate to the user the next step to be done (`start-env`)
3. Possibly, indicate to the user the following step to be done after `start-env`.

## Recommendation

Print a message once `new` is done, for example:
```
> create-comit-app new foo
Your project `foo` has been created!
Now, start your development environment by running `create-comit-app start-env`
```

### PR DESCRIPTION
**Pull request by n1rna**
Fixes #45.

### CODE REVIEW

File: src/new.rs [hunk 1]
```diff
@@ -1,3 +1,5 @@
+use crate::print_progress;
```
Comment (D4nte) [inline src/new.rs]: This is not needed, the `print_progress` macro is only useful when waiting for something that takes time.

File: src/new.rs [hunk 2]
```diff
@@ -21,4 +23,6 @@ pub fn new(name: String) {
 
     let _ = Repository::init(format!("./{}", name))
         .map_err(|e| panic!("Failed to create an empty Git repository: {:?}", e));
+    
+    print("Your project `{}` has been created!\nNow, start your development environment by running `create-comit-app start-env`", name)
```
Comment (D4nte) [inline src/new.rs]: Please use `println!`. Also, run `cargo fmt` to ensure the formatting is correct.

File: src/new.rs (hunk 1 above)
Comment (n1rna) [reply src/new.rs]: Oh, I forgot to delete this line.

**Comment by D4nte** [comment]
Can you please squash the commits in one?

**Pull request merged**